documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,45 +1,86 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. If your Raspberry Pi has both connectors, you can use the micro HDMI and LCD display outputs at the same time.
+
+Make sure your Raspberry Pi OS is up-to-date before you begin working with the Touch Display to reduce the risk of errors or driver issues.
 
 === Board Support
 
-The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
+The DSI display is designed to work with all models of Raspberry Pi, however early models without mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-sized stand-offs on the display PCB.
+
+=== Connecting a Raspberry Pi
+
+The following image shows how to attach a Raspberry Pi directly to the back of the Touch Display using its standoffs. It also shows the connections you need to make between Raspberry Pi and the display's PCB. You can also mount the touch display in a separate chassis if you have one available. The connections will remain the same, though you may need longer cables depending on the chassis you use.
+
+image::images/GPIO_power-500x333.jpg[DSI Display pinout]
+
+==== Connecting the display cable
+
+Connect one end of the ribbon cable to the `RPI-DISPLAY` port on the Touch Display PCB so the side with the blue stripe faces the display.
+
+// Close-up image here?
+
+Then connect the other end of the ribbon cable to the `DISPLAY` port on the Raspberry Pi. The blue stripe should face away from the Raspberry Pi.
+
+// Another close-up?
+
+If the ribbon cable isn't fully inserted, or it's not positioned correctly, you will experience issues with the display. You should always double-check this connection when troubleshooting, especially if you don't see anything on your display, or the display is showing a single color.
+
+==== Powering the Touch Display
+
+There are two options available to provide power to the Touch Display: using the Raspberry Pi's GPIO, or a separate power supply and micro USB cable. 
+
+===== Provide power with Raspberry Pi
```

Comment:
"Provide power with Raspberry Pi" -> "Provide power from the Raspberry Pi" ?
(perhaps it's just me, but "with" sounds strange here)

Reply:
Changed to "Provide power through the Raspberry Pi"